packages/common/driver-definitions/src/urlResolver.ts

```diff
@@ -93,7 +93,9 @@ export interface IDriverHeader {
 }
 
 declare module "@fluidframework/core-interfaces" {
-	// TODO: This interface should be documented. AB#7164
-	// eslint-disable-next-line @typescript-eslint/no-empty-interface, jsdoc/require-jsdoc
+	/**
+	 * Interface to represent headers to include in the container or data store load request.
```

Comment:
How is this interface intended to be used?

Reply:
Added